extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -231,6 +239,14 @@ export class DashboardWidget {
 		return view.modelBuilder.divContainer().withItems([buttonContainer]).component();
 	}
 
+	private setAutoRefresh(interval: SupportedAutoRefreshIntervals): void {
+		let classVariable = this;
```

Comment:
please do a test to see whether this is still running when the dashboard is closed.